package.json

```diff
@@ -39,6 +39,8 @@
     "eslint": "^3.14.0",
     "eslint-config-airbnb-base": "^11.0.1",
     "eslint-import-resolver-webpack": "^0.8.1",
+    "chart.js": "^2.4.0",
+    "dat.gui": "^0.6.1",
```

Comment:
dat.gui is used in GuiTools that does not use webpack (as it is an example) and in examples/index.html that also import it directly. What do you want exactly?